[2022-12-04 16:41:32,479] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-12-04 16:41:32,723] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-12-04 16:41:32,730] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:41:32,737] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 16:41:32,746] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:41:32,911] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-01-03 00:00:00+00:00
[2022-12-04 16:41:32,979] {standard_task_runner.py:52} INFO - Started process 802 to run task
[2022-12-04 16:41:33,122] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-01-03T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpgv_6wpvw', '--error-file', '/tmp/tmps7h678in']
[2022-12-04 16:41:33,161] {standard_task_runner.py:80} INFO - Job 119: Subtask world
[2022-12-04 16:41:33,907] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world scheduled__2022-01-03T00:00:00+00:00 [running]> on host fb90b48d1471
[2022-12-04 16:41:34,591] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-03T00:00:00+00:00
[2022-12-04 16:41:34,594] {logging_mixin.py:115} INFO - Airflow
[2022-12-04 16:41:34,596] {python.py:173} INFO - Done. Returned value was: None
[2022-12-04 16:41:34,813] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20220103T000000, start_date=20221204T164132, end_date=20221204T164134
[2022-12-04 16:41:34,963] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 16:41:35,158] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
